### PR TITLE
Updated Tables Documentation: Section Add and remove Rows

### LINKED ISSUE #708: Documentation: Adding Row to a table, dropping row and Adding row at index
I think Documentation on Rows is missing some basic operations such as how to add new Rows to a table, delete them and even adding rows at a certain index. It will certainly help someone who is interested in doing the same. 

Let me know if this would be helpful, so i can go ahead with a PR.

### PR DESCRIPTION
**Pull request by murtuza-ranapur**
Thanks for contributing.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

Changes in file userguide/tables.md

1. Added documentation to add new Row
2. Added documentation to add new Row at certain index
3. Added documentation to remove Row
4. Added documentation to remove Row based on some criteria

Fixes #708 
## Testing

Not required

### CODE REVIEW

File: docs/userguide/tables.md [hunk 1]
```diff
@@ -171,8 +171,51 @@ Tablesaw supports inner and outer joins between tables.
...
+newRow.setString("s","value1");
+newRow.setDate("d",LocalDate.now());
+Table newTableWithRowAtGivenIndex = top.append(bottom);
+```  
+
+###Remove rows from table
+
+To remove rows from table you need to call *dropRows()* on table with indexes that you want to drop
+```java
+Table newTableWithRemainingRows = table.dropRows(1,2,..);
```
Comment (ryancerf) [inline docs/userguide/tables.md]: Please remove the ,... so this would compile if copied and pasted.

File: docs/userguide/tables.md [hunk 2]
```diff
@@ -171,8 +171,51 @@ Tablesaw supports inner and outer joins between tables.
...
+```java
+Row newRow = aTable.appendRow();
+```
+then just set the values of each column in the row
+```java
+newRow.setString("s","value1");
+newRow.setDate("d",LocalDate.now());
+```
+
+###Adding row at certain index
```
Comment (ryancerf) [inline docs/userguide/tables.md]: I do not think this is a very common pattern and would prefer to leave this out of the docs. I think it is more noise than it is worth.

File: docs/userguide/tables.md [hunk 3]
```diff
@@ -171,8 +171,51 @@ Tablesaw supports inner and outer joins between tables.
 
 
 ## Add and remove rows
+###Adding New Row
 
+To add a new row into the table you simply need to call
```
Comment (ryancerf) [inline docs/userguide/tables.md]: This is a very common pattern. I can not believe we were missing this. Thanks for adding!

File: docs/userguide/tables.md (hunk 2 above)
Comment (murtuza-ranapur) [reply docs/userguide/tables.md]: i was also dubious about adding this point, but then i though it was certainly useful to me so it might be helpful to someone else too. Plus i also though it would give user idea of of how to split the tables (As the documentation already has section on combining the tables but not splitting). Anyway i will just leave this part out.

File: docs/userguide/tables.md (hunk 3 above)
Comment (murtuza-ranapur) [reply docs/userguide/tables.md]: Thanks @ryancerf actually i would like to contribute more in the documentations, so i think it would be great if i could get some guidelines for the same.

**Comment by murtuza-ranapur** [comment]
Made suggested changes and moved code to docs-src/main/userguide/tables.md

File: docs-src/main/userguide/tables.md [hunk 4]
```diff
@@ -171,8 +171,36 @@ Tablesaw supports inner and outer joins between tables.
...
+```
+then just set the values of each column in the row
+```java
+newRow.setString("s","value1");
+newRow.setDate("d",LocalDate.now());
+```
+
+###Remove rows from table
+
+To remove rows from table you need to call *dropRows()* on table with indexes that you want to drop
```
Comment (ryancerf) [inline docs-src/main/userguide/tables.md]: A bit wordy. Please change to something like.

To remove rows call *dropRows()* with indexes of the rows you want to drop.

Comment (murtuza-ranapur) [reply docs-src/main/userguide/tables.md]: Yeah sure

**Comment by murtuza-ranapur** [comment]
Yeah sure, meanwhile i will keep adding documentation in the missing sections.
